spiffworkflow-frontend/src/index.css

```
@@ -1,45 +1,36 @@
-/* site is mainly using white theme. */
-/* header is mainly using g100 */
-/* mockup wanted white, not grey, text */
+[data-theme='light'] {
+  --spiff-text-primary: #161616;
+  --spiff-text-secondary: #525252;
+  --spiff-text-muted: #777;
+  --spiff-surface: #fff;
+  --spiff-surface-muted: #f0f0f0;
+  --spiff-border: #ccc;
+  --spiff-error-text: #b71c1c;
+  color-scheme: light;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)